metadata-ingestion/src/datahub/classification/classifier_pipeline.py

```diff
@@ -0,0 +1,278 @@
+import base64
+import datetime
+import hashlib
+import logging
+import uuid
+from typing import Any, Dict, Iterable, List, Optional
+
+import click
+import requests
+
+from datahub.classification.classifier import ClassificationResult, Classifier
+from datahub.configuration.common import ConfigModel, DynamicTypedConfig
+from datahub.ingestion.api.common import PipelineContext, RecordEnvelope
+from datahub.ingestion.api.sampleable_source import SampleableSource
+from datahub.ingestion.api.source import Extractor
+from datahub.ingestion.api.transform import Transformer
+from datahub.ingestion.extractor.extractor_registry import extractor_registry
+from datahub.ingestion.source.source_registry import source_registry
+from datahub.ingestion.transformer.transform_registry import transform_registry
+from datahub.metadata.schema_classes import SchemaMetadataClass
+
+logger = logging.getLogger(__name__)
+
+
+class SourceConfig(DynamicTypedConfig):
+    extractor: str = "generic"
+
+
+class ClassifierPipelineConfig(ConfigModel):
+    datahub_base_url: str
+    datahub_username: str
+    datahub_encrypted_password: str
+    ddb_client: Any  # Type hints not available for boto3 clients
```

Comment:
Are config models supposed to have these python types? I thought `ConfigModel` is supposed to hold config values like dicts strs booleans etc

Reply:
Yeah when I was working on the metrics client I pulled these out and made them passed into the constructor. Definitely looks confusing calling them config

Reply:
I already have this changed in the metrics commit based on this, so I'll make this change in that PR